rebar.config

```diff
@@ -62,7 +62,6 @@
 
   %%% Observability
   {telemetry, "1.2.0"},
-  {lager, "3.9.2"}, % We're keeping this to override the dependencies
```

Comment:
This wasn't removed from the lock file, wasn't it unlocked or is there some other dep still using it? 🤔

Reply:
Apparently I just left it there somehow. It seems I need to scan the MIM repo once more for lager leftovers, as I did for the `nk*` repos.